scripts/emod3d_scripts/__init__.py

```diff
@@ -0,0 +1,15 @@
+import argparse
```

Comment:
Would be nice to include as part of the docstring how this works.

Reply:
Ah, that was my pycharm template dropping itself in where it doesn't belong. Will remove.